laravel.md

```diff
@@ -0,0 +1,52 @@
+# Laravel
+
+**Prerequisite knowledge:** php, oop, mysql, html, frameworks, mvc, composer, dependency managment
+
+## Intro
+Laravel is a free, open-source PHP web framework, created by Taylor Otwell and intended for the development of web applications following the model–view–controller (MVC) architectural pattern and based on Symfony. Some of the features of Laravel are a modular packaging system with a dedicated dependency manager, different ways for accessing relational databases, utilities that aid in application deployment and maintenance, and its orientation toward syntactic sugar. [[Source](https://en.wikipedia.org/wiki/Laravel)]
+
+## Classification / acceptance questions
+
+### Apprentice aka _My First TODO app_
+* I know what Laravel is
+* I can use Laravel router to interact with the end user
+* I can create simple blade templates to separate presentation layer from business logic
+* I can Create, Read, Update, Delete entries in DB through Eloquent Model 
+* I can use MVC design pattern in Laravel
```

Comment:
This sounds very generic, can you add some examples or split it into more specific points?

Reply:
This is a summary of 3 points above. I'm not sure if I can phrase an example here that will be understood by Aprentice. Maybe it's better to remove it?
On the other hand I think it's important for a person that starts his journey with frameworks to know that MVC is important part of his life.